docs/content/reference/install-configuration/providers/others/file.md

```diff
@@ -0,0 +1,221 @@
+      service = "service-foo"
+      rule = "Path(`/foo`)"
+
+  # Add the middleware
+  [http.middlewares]
+    [http.middlewares.my-basic-auth.basicAuth]
+      users = ["test:$apr1$H6uskkkW$IgXLP6ewTrSuBkTrqE8wj/",
+                "test2:$apr1$d9hr9HBB$4HxwgUir3HP4EsggP/QNo0"]
+      usersFile = "etc/traefik/.htpasswd"
+
+  # Add the service
+  [http.services]
+    [http.services.service-foo]
+      [http.services.service-foo.loadBalancer]
+        [[http.services.service-foo.loadBalancer.servers]]
+          url = "http://foo/"
+        [[http.services.service-foo.loadBalancer.servers]]
+          url = "http://bar/"
+```
+
+## Configuration Options
+
+| Field | Description                                               | Default              | Required |
+|:------|:----------------------------------------------------------|:---------------------|:---------|
+| `providers.providersThrottleDuration` | Minimum amount of time to wait for, after a configuration reload, before taking into account any new configuration refresh event.<br />If multiple events occur within this time, only the most recent one is taken into account, and all others are discarded.<br />**This option cannot be set per provider, but the throttling algorithm applies to each of them independently.** | 2s  | No |
+| `providers.file.filename` | Defines the path to the configuration file.  |  ""    | Yes   |
+| `providers.file.directory` | Defines the path to the directory that contains the configuration files. The `filename` and `directory` options are mutually exclusive. It is recommended to use `directory`.  |  ""    | Yes   |
+| `providers.file.watch` | Set the `watch` option to `true` to allow Traefik to automatically watch for file changes. It works with both the `filename` and the `directory` options. | true | No |
+
+!!! warning "Limitations"
+
+    With the file provider, Traefik listens for file system notifications to update the dynamic configuration.
+
+    If you use a mounted/bound file system in your orchestrator (like docker or kubernetes), the way the files are linked may be a source of errors.
+    If the link between the file systems is broken, when a source file/directory is changed/renamed, nothing will be reported to the linked file/directory, so the file system notifications will be neither triggered nor caught.
+
+    For example, in Docker, if the host file is renamed, the link to the mounted file is broken and the container's file is no longer updated.
+    To avoid this kind of issue, it is recommended to:
+
+    * set the Traefik [**directory**](#directory) configuration with the parent directory
+    * mount/bind the parent directory
+
+    As it is very difficult to listen to all file system notifications, Traefik uses [fsnotify](https://github.com/fsnotify/fsnotify).
+    If using a directory with a mounted directory does not fix your issue, please check your file system compatibility with fsnotify.
+
+### Go Templating
```

Comment:
WDYT to move this section into the dynamic configuration section? It's not part of the provider configuration IMO

Reply:
That makes sense, will do!